_pages/how-we-work/networks.md

```diff
@@ -1,13 +0,0 @@
----
-title: Networks
----
-
-Learn how to connect to the GSA network on campus and remotely.
-
-[GSA Network Connection Guide](https://docs.google.com/document/d/1nBNXt6Ov4KWmpz6y9rgKw93mxZucVsoYC4PFABTeIA4/edit#heading=h.6hzaqzho5cc3)
-
-----
-
-#### Still have questions?
-
-Ask in Slack: [#it-issues](https://gsa-tts.slack.com/messages/questions/) or [#distributed](https://gsa-tts.slack.com/messages/distributed/)
```

Comment:
Can we make this page redirect, just in case?

Reply:
It's wiped out, and our CircleCI tests ensure there are no remaining links to it. In the interest of cleaning up after myself (and because I'm not a fan of client-side redirects) I'd rather just remove it.